fedora/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM fedora:26
 LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+LABEL name "Linuxbrew/fedora26"
```

Comment:
Should this one be named `Linuxbrew/fedora` for consistency with the directory name?

Reply:
ooops